### PR TITLE
[zh] Update custom-resources.md

### PR DESCRIPTION
**Pull request by ptux**
- Make it clear that the main API server delegates request to the user-created API server.

- Explanation for API aggregation, not standalone API server.

Related PRs: ​
https://github.com/kubernetes/website/pull/30405

### CODE REVIEW

**Comment by tengqm** [comment]
/approve
/hold
Feel free to lift the hold when the upstream change is in.

**Comment by jihoon-seo** [comment]
The English PR #30405 is merged on 2021-11-16 (13 days ago).

File: content/zh/docs/concepts/extend-kubernetes/api-extension/custom-resources.md [hunk 1]
```diff
@@ -327,9 +327,9 @@ Kubernetes API 主服务器能够处理诸如 *pods* 和 *services* 这些内置
 按通用的方式通过 [CRD](#customresourcedefinitions) 来处理定制资源。
 
 [聚合层（Aggregation Layer）](/zh/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation/)
-使得你可以通过编写和部署你自己的独立的 API 服务器来为定制资源提供特殊的实现。
-主 API 服务器将针对你要处理的定制资源的请求全部委托给你来处理，同时将这些资源
-提供给其所有客户。
+使得你可以通过编写和部署你自己的 API 服务器来为定制资源提供特殊的实现。
+主 API 服务器将针对你要处理的定制资源的请求全部委托给你自己的 API 服务器来处理，同时将这些资源
```
Comment (tengqm) [inline content/zh/docs/concepts/extend-kubernetes/api-extension/custom-resources.md:331]: The English text in line 318-321 needs to be synced.

**Pull request merged**